\documentclass[25pt, a0paper, 
portrait, margin=0mm, innermargin=10mm,
blockverticalspace=15mm, colspace=15mm,
subcolspace=8mm]{tikzposter}

\usepackage{graphicx}
\usepackage{booktabs}
\usepackage{natbib}

\tikzposterlatexaffectionproofoff

\settitle{ \centering \vbox{
     \@titlegraphic \\[\TP@titlegraphictotitledistance] \centering
     \color{titlefgcolor} {\bfseries \Huge  \@title \par}
     \vspace*{2em}
     {\Large \@author \par} \vspace*{0em} {\LARGE \@institute}
}}


\title{\parbox{\linewidth}{\centering Barking up the Right Trees: the
    Influence of Forest Fire on the Genetic Architecture of Pine
    Populations in the Southeastern United States}}

%\institute{Virginia Commonwealth University}

\author{PI: Christopher J. Friedline, Ph.D. \\ Sponsor: Andrew J. Eckert, Ph.D.}

\titlegraphic{
 \includegraphics[scale=0.8]{vcu_nsf}
}


\definecolor{vcugold}{RGB}{248,184,0}

\definecolorstyle{cjfStyle} {
     \definecolor{colorOne}{named}{white}
     \definecolor{colorTwo}{named}{black}
     \definecolor{colorThree}{named}{vcugold}
 }{
     % Background Colors
     \colorlet{backgroundcolor}{colorOne}
     \colorlet{framecolor}{black}
     % Title Colors
     \colorlet{titlefgcolor}{vcugold}
     \colorlet{titlebgcolor}{colorTwo}
     % Block Colors
     \colorlet{blocktitlebgcolor}{colorThree}
     \colorlet{blocktitlefgcolor}{black}
     \colorlet{blockbodybgcolor}{white}
     \colorlet{blockbodyfgcolor}{black}
     % Innerblock Colors
     \colorlet{innerblocktitlebgcolor}{white}
     \colorlet{innerblocktitlefgcolor}{black}
     \colorlet{innerblockbodybgcolor}{colorThree!30!white}
     \colorlet{innerblockbodyfgcolor}{black}
     % Note colors
     \colorlet{notefgcolor}{black}
     \colorlet{notebgcolor}{colorThree!20!white}
     \colorlet{noteframecolor}{colorTwo}
}

\usetheme{Default}\usecolorstyle{cjfStyle}

\begin{document}

\maketitle[width=0.972\textwidth, titletextscale=5]

\block{Project summary}{\Large The main goal of this project is to
  investigate bark thickness as a \textbf{fire-adapted phenotype}
  \citep{He:NewPhytol:2012} in four biologically and
  economically-important species of pines ranging along the
  eastern and south-eastern United States. The \textbf{evolution} of
  this \textbf{complex trait} will be studied in populations of slash
  pine (\textit{Pinus elliottii}), pond pine (\textit{P.\ serotina}),
  loblolly pine (\textit{P.\ taeda}), and long leaf pine (\textit{P.\
    palustris}). Next generation sequencing (NGS) of 15 individuals
  from 20 \textbf{natural populations} for each species ($15 \times 20 \times 4
  = 1200$ total individuals) will be used to genotype individuals with
  the goal of uncovering \textbf{shared genetic architecture} of these
  four closely related species. The results of this research, while
  providing new genomic resources for \textbf{non-model species} and
  augmenting existing resources, will address fundamental principles
  in evolutionary biology and serve to inform and improve breeding
  programs and land management initiatives.}

\begin{columns}
  \column{0.5}
  \block{Objectives}{
    \begin{itemize}
      \large
    \item Field sampling: needle collection, bark thickness,
      DBH, height, GPS (54\%)
    \item Bioinformatic pipelines and code development: GitHub (99.9\%)
    \item DNA library preparation \citep{Parchman:MolEcol:2012,Peterson:PlosOne:2012} (5\%)
    \item DNA Sequencing (HiSeq 2500)
    \item GBS vs. \textit{P. taeda} genome (v1.01, 14.4M scaffolds)
    \end{itemize}
  }

  \block{Hypothesis testing} {
    \large
    \begin{enumerate}
    \item What is the genetic architecture of bark thickness for the four focal pine species that are adapted to fire?
    \item Are candidate SNPs shared across the four focal species, and
      if so, are they shared to a greater degree than randomly sampled
      SNPs from the controls?
    \item Are reconstructed allele frequencies at shared candidate SNPs
      correlated more so with reconstructed bark thickness than shared
      control SNPs across the phylogeny for these four species?
    \item To what extent is natural selection affecting rates of molecular evolution at candidate genes?
    \end{enumerate}
  }

  \block{Project status}{
    \begin{tikzfigure}
      \includegraphics[width=0.427\textwidth]{map.png} 
    \end{tikzfigure}
     \begin{center}
      Figure 1: Sampled locations marked by triangle, color
      indicates species. Overlays are  the natural ranges of the
      populations. \\(blue=\textit{P.\ taeda}, yellow=\textit{P.\
        palustris}, green=\textit{P.\ serotina}, magenta=\textit{P.\
        elliottii})
    \end{center}    
    
    \innerblock[]{}{
    \begin{center}
      Table 1: Summary statistics for four \textit{Pinus} species; $r$
      values are correlations with average bark thickness for all
      individuals per species.
    \end{center}


      \begin{tikzfigure}
      \centering
      \begin{tabular}{lrrrrrr}
        \toprule
        Species & N & Pops & DBH (cm) & $r_{DBH}$ & Height (m) & $r_{Height}$\\
        \midrule
        \textit{P.\ elliottii} & 55 & 3 & 39.5 (10.0) & 0.5 & 24.8 (5.1) & 0.04 \\ 
        \textit{P.\ palustris} & 297 & 18 &  44.2 (8.2) & 0.36 & 22.9 (4.2) & -0.08 \\
        \textit{P.\ serotina} & 23 & 1 & 29.4 (7.8) & 0.02 & 20.2 (6.2) & 0.02 \\
        \textit{P.\ taeda} & 370 & 21 & 47.5 (12.9) & 0.46 & 27.0 (6.2) & 0.21 \\
        \bottomrule
      \end{tabular}
    \end{tikzfigure}
  } 
}

  \column{0.5}

  \block{Preliminary results}{
    \innerblock[]{}{
    \begin{tikzfigure}
      \includegraphics[width=0.2\textwidth]{minted_r-crop}
    \end{tikzfigure}
    }

    \vspace*{0.6em}
   

    \begin{center}
      Table 2: Nested linear mixed model testing for effect 
      of population on bark thickness.
    \end{center}
     
     \begin{tikzfigure} 
      \centering
      \begin{tabular}{lrrrrrrr}
        \toprule
        & \multicolumn{2}{c}{Model 1} & \multicolumn{2}{c}{Model 2} \\
        \cmidrule(l){2-3} \cmidrule(l){4-5}
        Species & $lnL$ & $\sigma^2_{Name/Pop/Resid}$ & $lnL$ & $\sigma^2_{Name/Resid}$ & $\chi^2$ & $p$\\ 
        \midrule
        \textit{P.\ elliottii} & 76.4 & 0.039/0.000/0.016 & 76.4 & 0.039/0.000/0.016 & 0 & 1\\
        \textit{P.\ palustris} & 593.7 & 0.018/0.005/0.013 & 577.8 & 0.023/0.013 & 31.7 & $\mathbf{1.8e^-8}$ \\
        \textit{P.\ serotina} & & & 52.4 & 0.036/0.009 & & \\
        \textit{P.\ taeda} & -693.9 & 0.039/0.010/0.120 & -706.3 & 0.049/0.120 & 24.7 & $\mathbf{6.6e^-7}$ \\
        \bottomrule
      \end{tabular}
    \end{tikzfigure}
   
    \vspace*{1.91em}
   

    \innerblock[]{}{
    \begin{tikzfigure}
      \includegraphics[width=0.25\textwidth]{minted_mantel-crop}
    \end{tikzfigure}
    }

    \vspace*{0.6em}
   

    \begin{center}
    Table 3: Significant Mantel correlations ($p < 0.05$) for averaged and 
    individual bark thickness measurements to BIOCLIM variables \citep{citeulike:1113062}*. All Mantel
    tests between bark thickness and lat/lon were not significant.
    \end{center}


    \begin{tikzfigure} 
      \centering
      \begin{tabular}{lrrrrrrrr}
        \toprule
        &  \multicolumn{2}{c}{Averaged} & \multicolumn{6}{c}{Individual}\\
        \cmidrule(l){2-3} \cmidrule(l){4-9}
        Species & Bio10 & Bio12 & Bio2 & Bio7 & Bio9 & Bio12 & Bio13 & Bio16 \\
        \midrule
        \textit{P.\ elliottii} & & 0.47 & & & & & & \\
        \textit{P.\ palustris} & & & 0.07 & 0.05 & 0.05 & 0.07 & 0.04 & 0.05\\
        \textit{P.\ serotina} & & & & & & & & \\
        \textit{P.\ taeda} & 0.24 & & & & & & \\
        \bottomrule
      \end{tabular}
    \end{tikzfigure}
    
    \begin{center}
      \small
      *Mean diurnal range (2), Temp.\ annual range (7),
      Mean temp.\ of driest quarter (9), Annual precip.\ (12), \\
      Precip.\ of wettest month (13), Precip.\ of wettest quarter (16). 
    \end{center}
}

  
  \block{Preliminary insights} {
    \begin{itemize}\Large
    \item There is evidence of population differentiation of the thick
      bark phenotype. \\(Table 2).
    \item Environmental correlations are present but may currently
      suffer from under-sampling or other bias (Table 3).
    \item Though climate is structured spatially, the phenotype
      appears not to be; historical demography is unlikely confounding
      the results (Table 3).
    \end{itemize}
  }
  
  \block{References} {
      \footnotesize
      \renewcommand{\refname}{\vspace{-1.25em}} 
      \bibliographystyle{sysbio}
      \bibliography{refs}
  }

\end{columns}


\end{document}
