\documentclass[25pt, a0paper, portrait, margin=0mm, innermargin=15mm,
blockverticalspace=15mm, colspace=15mm,
subcolspace=8mm]{tikzposter}

\usepackage{graphicx}

\tikzposterlatexaffectionproofoff

\title{\parbox{\linewidth}{\centering Barking up the Right Trees: the
    Influence of Forest Fire on the Genetic Architecture of Pine
    Populations in the Southeastern United States}}

\institute{Virginia Commonwealth University}

\author{Christopher J. Friedline, Ph.D. \\ PI: Andrew J. Eckert, Ph.D.}

\usetheme{Default}

\begin{document}

\maketitle

\block{Project Summary}{The main goal of this project is to
  investigate bark thickness as a fire-adapted phenotype in four,
  economically-important, species of pines with a range along the
  eastern and south-eastern United States. The evolution of this
  complex trait will be studied in populations of slash pine
  (\textit{Pinus elliottii}), pond pine (\textit{P.\ serotina}),
  loblolly pine (\textit{P.\ taeda}), and long leaf pine
  (\textit{P.\ palustris}). Next generation sequencing (NGS) of 15
  individuals from 20 natural populations for each species ($15 \times
  20 \times 4 = 1200$ total individuals) will be used to genotype
  individuals with the goal of uncovering shared genetic architecture
  of these four closely related species. The results of this research,
  while providing new genomic resources for non-model species and
  augmenting exiting resources, will address fundamental principles in
  evolutionary biology and serve to inform and improve breeding
  programs and land management initiatives.  }

\begin{columns} % See Section 4.4
  \column{0.5} % See Section 4.4
  \block{Objectives}{
    \begin{itemize}
    \item Field sampling: needle collection, bark thickness,
      DBH, height, GPS
    \item DNA library preparation (Parchman)
    \item DNA Sequencing (HiSeq 2500)
    \item GBS vs. \textit{P. taeda} genome (v1.01, 14.4M scaffolds)
    \end{itemize}
  }

  \block{Hypothesis Testing} {
    \begin{enumerate}
    \item What is the genetic architecture of bark thickness for the four focal pine species that are adapted to fire?
    \item Are candidate SNPs shared across the four focal species, and
      if so, are they shared to a greater degree than randomly sampled
      SNPs from the controls?
    \item Are reconstructed allele frequences at shared candidate SNPs
      correlated more so with reconstructed bark thickness than shared
      control SNPs across the phy- logeny for these four species?
    \item To what extent is natural selection affecting rates of molecular evolution at candidate genes?
    \end{enumerate}
  }

  \block{Project status}{
    \begin{tikzfigure}[Sampled locations marked by triangle, color
      indicates species. Overlay is the natural range of the
      populations \\ (blue=\textit{P.\ taeda}, yellow=\textit{P.\
        palustris}, green=\textit{P.\ serotina}, magenta=\textit{P.\
        elliotti})]
      \includegraphics[width=0.4\textwidth]{map.png} 
    \end{tikzfigure}
  }

  \note[targetoffsetx=50mm, targetoffsety=-27mm, width=8.5cm, roundedcorners=10] {
    \footnotesize Species (ind/pop)
    \begin{itemize}
    \item \textit{P.\ elliotti} (55/3)
    \item \textit{P.\ palustris} (297/18)
    \item \textit{P.\ serotina} (23/1)
    \item \textit{P.\ taeda} (370/21)
    \end{itemize}
  }

  \column{0.5}

  \block{References}{
    
  }
  
  \block{Contact}{
    \begin{itemize}
    \item e-mail: cfriedline@vcu.edu
    \item twitter: @noituloveand
    \item web: http://chris.friedline.net
    \item github: @cfriedline
    \end{itemize}
    
  }

\end{columns}


\end{document}
